src/Components/Messages.jsx

```diff
@@ -1,5 +1,16 @@
 import React, { Component } from 'react';
 
-const Messages = (props) => <div><ul></ul></div>
+const Messages = ({messages}) => {
 
-export default Messages;
\ No newline at end of file
+    const messageLog = messages.map ( (message) => {
+        return ( <p>{message}</p>)
+    });
+
+    return (
```

Comment:
Might be a bit nicer to merge the messageLog function into the JSX, maybe? 

```JavaScript
const Messages = ({messages}) => {
    return (
        <div>
            {messages.map((message) => <p>{message}</p>)}
        </div>
    );
}
```

Reply:
Up to you, I usually favour fewer lines of code, but if you think having messageLog seperate is more readable feel free to keep that.

Reply:
This looks tidier and more logical, pushed change